zipline/data/data_portal.py

```diff
@@ -262,15 +251,17 @@ def _get_ctable(self, asset):
             else:
                 path = "{0}/{1}.bcolz".format(self._minutes_futures_path, sid)
         elif isinstance(asset, Equity):
-            if self._equity_sid_path_func is not None:
-                path = self._equity_sid_path_func(
-                    self._minutes_equities_path, sid
+            if self._equity_minute_reader.sid_path_func is not None:
+                path = self._equity_minute_reader.sid_path_func(
+                    self._equity_minute_reader.rootdir, sid
                 )
             else:
-                path = "{0}/{1}.bcolz".format(self._minutes_equities_path, sid)
+                path = "{0}/{1}.bcolz".format(
+                    self._equity_minute_reader.rootdir, sid)
 
         else:
-            path = "{0}/{1}.bcolz".format(self._minutes_equities_path, sid)
+            path = "{0}/{1}.bcolz".format(
```

Comment:
This may be a silly question, but if the data portal is now passed a bcolz reader, why does it also need to understand bcolz and where to read from?  I would think that's abstracted away by the reader.